docs/en/deployment-guides/horizontal-scaling.md

```diff
@@ -0,0 +1,468 @@
+---
+slug: /en/architecture/horizontal-scaling
+sidebar_label: Scaling out
+sidebar_position: 10
+title: Scaling out
+---
+import ReplicationShardingTerminology from '@site/docs/en/_snippets/_replication-sharding-terminology.md';
+import ConfigFileNote from '@site/docs/en/_snippets/_config-files.md';
+
+
+## Description
+This example architecture is designed to provide scalability.  It includes three nodes: two combined ClickHouse plus coordination (ClickHouse Keeper) servers, and a third server with only ClickHouse Keeper to finish the quorum of three. With this example, we'll create a database, table, and a distributed table that will be able to query the data on both of the nodes.
+
+## Level: Basic
+
+<ReplicationShardingTerminology />
+
+## Environment
+### Architecture Diagram
+![Architecture diagram for 2 shards and 1 replica](@site/docs/en/deployment-guides/images/scaling-out-1.png)
+
+|Node|Description|
+|----|-----------|
+|chnode1|Data + ClickHouse Keeper|
+|chnode2|Data + ClickHouse Keeper|
+|chnode3|Used for ClickHouse Keeper quorum|
+
+:::note
+In the more advanced configurations ClickHouse Keeper will be run on separate servers.  This basic configuration is running the Keeper functionality within the ClickHouse Server process.  As you scale out you may decide to separate the ClickHouse Servers from the Keeper servers.  The instructions for deploying ClickHouse Keeper standalone are available in the [installation documentation](/docs/en/getting-started/install.md/#install-standalone-clickhouse-keeper).
+:::
+
+## Install
+
+Install Clickhouse on three servers following the [instructions for your archive type](/docs/en/getting-started/install.md/#available-installation-options) (.deb, .rpm, .tar.gz, etc.). For this example, you will follow the installation instructions for ClickHouse Server and Client on all three machines.       
+
+## Editing configuration files
+
+<ConfigFileNote />
+
+## chnode1 configuration
+
+For chnode1 there are five configuration files.  You may choose to combine these files into a single file, but for clarity in the documentation it may be simpler to look at them separately.  As you read through the configuration files you will see that most of the configuration is the same between chnode1 and chnode2; the differences will be highlighted.
```

Comment:
```suggestion
For chnode1, there are five configuration files.  You may choose to combine these files into a single file, but for clarity in the documentation it may be simpler to look at them separately.  As you read through the configuration files, you will see that most of the configuration is the same between chnode1 and chnode2; the differences will be highlighted.
```

Just editing according to what Grammarly suggested :)